docs/topics/about-tools.adoc

```diff
@@ -1,14 +1,20 @@
 // Module included in the following assemblies:
 //
-// * docs/cli-guide/master.adoc
+// * docs/getting-started-guide/master.adoc
 
 [id="about-tools_{context}"]
 = About the tools
 
-The {ProductName} ({ProductShortName}) provides several tools to assist you in the various stages of your migration and modernization efforts. Review the details of each tool to determine which is right for your project.
+You can use any of these {ProductName} ({ProductShortName}) tools to help you migrate and modernize projects:
 
-* CLI
-* Web console
-* {PluginName} for Eclipse and Red Hat CodeReady Studio
-* {ProductShortName} extension for Visual Studio Code
-* {MavenName}
+* link:{ProductDocUserGuideURL}[CLI]: The {ProductShortName} command-line tool lets you assess and prioritize migration and modernization efforts for applications without the overhead of a graphical user interface. The {CLIName} includes customization options and integrates with external automation tools.
+* link:{ProductDocWebConsoleGuideURL}[Web console]: The {ProductShortName}  Web console lets you easily assess and prioritize migration and modernization efforts by grouping applications into projects for analysis.
```

Comment:
Remove extra space. Rewrite "lets".

If we follow our normal practices, it should be "Group applications into projects for analysis by using the MTA web console."